submissions/semenjob/style.css

```diff
@@ -0,0 +1,141 @@
+* {
```

Comment:
🔥

Reply:
Is this a smiley like it's good or bad?)

Reply:
good